### PR TITLE
Av 651 xblr taxonomy to resource

### PR DESCRIPTION
**Pull request by Aaronkala**
- Added maturity label to resource view.
- Improved resource view styles and layout.

### CODE REVIEW

File: modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less [hunk 1]
```diff
@@ -151,8 +151,8 @@
     .actions {
         position: relative;
         float: right;
-        top: -10px;
-        right: -15px;
+        top: 0px;
+        right: 0px;
```
Comment (Zharktas) [inline modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less]: upstream_ckan should not be modified as the modifications are lost when ckan is updated.

Comment (Aaronkala) [reply modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less]: Ok, moved them to a different file

File: modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less [hunk 2]
```diff
@@ -151,8 +151,8 @@
     .actions {
         position: relative;
         float: right;
-        top: -10px;
-        right: -15px;
+        top: 15px;
+        right: 15px;
```
Comment (Zharktas) [inline modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less]: Still different from the original, this file should not have any modifications.

Comment (Aaronkala) [reply modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less]: Sorry, I don't know what happened in my brain.

**Pull request merged**